ouroboros-consensus-cardano-tools/test/disk/config/genesis-conway.json

```diff
@@ -0,0 +1 @@
+{ "genDelegs": {} }
```

Comment:
TODO another piece of the theme: does the re-scoped Conway still need a genesis file?